src/librustc_resolve/lib.rs

```diff
@@ -2354,14 +2360,10 @@ impl<'a> Resolver<'a> {
                 }
             }
 
-            // Try Levenshtein if nothing else worked.
-            if let Some(candidate) = this.lookup_typo_candidate(path, ns, is_expected) {
-                err.span_label(ident_span, &format!("did you mean `{}`?", candidate));
-                return err;
-            }
-
             // Fallback label.
-            err.span_label(base_span, &fallback_label);
+            if err.span.span_labels().is_empty() {
```

Comment:
@Freyskeyd 
IIRC, `span_labels().is_empty()` had quirks and didn't work as expected.
From the tests it looks like it still doesn't work.
You can just set some boolean flag if Levenshtein worked and check it here.